boards/arm/mimxrt1050_evk/mimxrt1050_evk_qspi.dts

```diff
@@ -11,17 +11,19 @@
 #if defined(CONFIG_CODE_ITCM)
 		zephyr,flash = &itcm0;
 #elif defined(CONFIG_CODE_QSPI)
-		zephyr,flash = &qspi0;
+		zephyr,flash = &is25wp064;
```

Comment:
Why did you change this for only one board?

Reply:
Mistake, I will fix.